### DISCUSSION TITLE
Still Have Not Received Benefits Of GitHub Education

### DISCUSSION THREAD

**Discussion by SlimeCEO** [community/community#145216] [GitHub Education]
### Hi everyone,

I am writing regarding my recent approval for GitHub Education benefits. While my request has been approved for three days now, I have yet to receive access to my academic benefits.

I have submitted my application multiple times and ensured all required documentation was correct. Despite this, the benefits have not been activated in my account.

Could you please help resolve this issue or provide guidance on the next steps? I would greatly appreciate any assistance you can offer to expedite this process.

**Comment by davevad93** [comment]
Hi @SlimeCEO ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @SlimeCEO was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @SlimeCEO was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287
